src/Symfony/Component/Serializer/ByteOrderMark.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer;
+
+final class ByteOrderMark
+{
+    /**
+     *  UTF-8 BOM sequence.
+     */
+    const BOM_UTF8 = "\xEF\xBB\xBF";
```

Comment:
I think we should not handle any other BOM.
The encoder/decoder doesn't work at all with non-ASCII encodings anyway.